stylesheets/components/badges/badges.css

```diff
@@ -0,0 +1,83 @@
+@import "/stylesheets/index.css";
+@import "/stylesheets/components/avatars/avatars.css";
+
+
+.general-badges {
+    margin-bottom: 0;
+}
+
+.badges-on-icons {
+    background: var(--badges-on-icons-section);
+}
+
+.icon-with-badge {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.badge-icon-showcase {
+    position: relative;
+}
+
+.badge {
+    background: var(--badge-default-bg);
+    height: 1.25rem;
+    width: 1.25rem;
+    border-radius: 50%;
+    color: var(--badge-default-text);
+    text-align: center;
+    position: absolute;
+    top: -5px;
+    right: -6px;
+}
+
+.icon-badge{
+    font-size: small;
+    padding: 1px;
+}
+.badges-on-avatars {
+    background: var(--badges-on-avatars-section);
+}
+
+.avatar-with-badge{
+    display: flex;
+    justify-content: space-evenly;
+    padding: 1rem;
+}
+
+.avatar-badge{
+    display: flex;
+    position: relative;
+}
+
+.av-badge{
+    right: 1px;
```

Comment:
Give them different name because both looks same.

Reply:
Thank you for pointing this out. Done.